wikimedia/wikipedia-ios
# T377899 edit spanish

**Phabricator:** 
https://phabricator.wikimedia.org/T377899

### Notes
* Hardcoded language check

### Test Steps
1. Set device language/app language to Spanish
2. Navigate to YIR 
3. Go to the editing slide
4. Click the link, ensure it goes to the Spanish editing section

### Screenshots/Videos


https://github.com/user-attachments/assets/969be858-29f9-4fd1-9938-e5ecc4c6efb9
